apps/etl-func/avro/message.json

```diff
@@ -0,0 +1,141 @@
+      "doc": "An ULID that identifies the message inside the message collection.",
+      "type": "string"
+    },
+    {
+      "name": "sender_service_id",
+      "doc": "The id of the service that sent the message.",
+      "type": "string"
+    },
+    {
+      "name": "sender_user_id",
+      "doc": "The id of the user that sent the message.",
+      "type": "string"
+    },
+    {
+      "name": "is_pending",
+      "doc": "A boolean that indicates if the message has been processed, is_pending equals false indicates that the message has not been processed yet.",
+      "type": "boolean"
+    },
+    {
+      "name": "content_subject",
+      "doc": "The subject of the message.",
+      "type": "string"
+    },
+    {
+      "name": "content_type",
+      "doc": "Indicates the type of the message.",
+      "type": [
+        "null",
+        {
+          "type": "enum",
+          "name": "MessageContentType",
+          "symbols": [
+            "GENERIC",
+            "PAYMENT",
+            "EU_COVID_CERT",
+            "SEND",
+            "PAGOPA_RECEIPT"
+          ]
+        }
+      ],
+      "default": null
+    },
+    {
+      "name": "content_payment_data_amount",
+      "doc": "An integer that indicates the amount of the payment.",
+      "type": "int",
+      "default": 0
+    },
+    {
+      "name": "content_payment_data_notice_number",
+      "doc": "An id that identifies the the payment.",
+      "type": ["null", "string"],
+      "default": null
+    },
+    {
+      "name": "content_payment_data_invalid_after_due_date",
+      "doc": "A boolean that indicates if the payment is considered invalid after the due date.",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "content_payment_data_payee_fiscal_code",
+      "doc": "The fiscal code of the payee.",
+      "type": ["null", "string"],
+      "default": null
+    },
+    {
+      "name": "require_secure_channels",
+      "doc": "A boolean that indicates whether the message contains sensitive informations. If this value is true the push notification sent is anonymous and the email notification will not be sent.",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "timestamp",
+      "doc": "A timestamp that indicates when the message was created.",
+      "type": "long",
+      "logicalType": "timestamp-millis",
+      "default": 0
+    },
+    {
+      "name": "feature_level_type",
+      "doc": "A field that indicates whether the message is PREMIUM or not.",
+      "type": {
+        "type": "enum",
+        "name": "MessageFeatureLevelType",
+        "symbols": ["STANDARD", "ADVANCED"]
+      },
+      "default": "STANDARD"
+    },
+    {
+      "name": "anonymous_fiscal_code",
+      "doc": "Tokenized fiscal code of the citizen.",
+      "type": "string"
+    },
+    {
+      "name": "configuration_id",
+      "doc": "An ULID that identifies the remote content configuration.",
+      "type": "string"
+    },
```

Comment:
why is this value needed? I don't think that DL will use RC configurations

Reply:
Removed [here](https://github.com/pagopa/io-messages/pull/40/commits/bac56ac4a53445ae6bcaee77c71a10ff5590b188)